tls/s2n_server_key_exchange.h

```diff
@@ -27,6 +27,9 @@ int s2n_dhe_server_key_recv_parse_data(struct s2n_connection *conn, union s2n_ke
 int s2n_ecdhe_server_key_recv_parse_data(struct s2n_connection *conn, union s2n_kex_raw_server_data *raw_server_data);
 int s2n_kem_server_key_recv_parse_data(struct s2n_connection *conn, union s2n_kex_raw_server_data *raw_server_data);
 
+int s2n_dhe_server_key_external(struct s2n_connection *conn, struct s2n_blob *data_to_sign);
+int s2n_ecdhe_server_key_external(struct s2n_connection *conn, struct s2n_blob *data_to_sign);
```

Comment:
i'm struggling to grok what `external` means in this cases as a naive reader of the code.  can you consider a more meaningful name?